### PR TITLE
45 change team of user

### PR DESCRIPTION
**Pull request by dominic-heil**
This PR implements a way to add users to the "Admins" team if the current user in the frontend is in the Admins team.

The admin can search for a user by their mail, where they can invite the user to the Admins team or remove them if they are part of the Admins team. 

image

To test this fully, appwrite must be able to send emails. Without a setup email, the invitation can be seen in the appwrite UI as "pending".

### CODE REVIEW

File: frontend/src/pages/EmailConfirmPage.js [hunk 1]
```diff
@@ -19,9 +19,7 @@ export default function EmailConfirmPage() {
 			if (res){
 				setWasConfirmed(true);
 			}
-		}).catch(r =>
-			console.log(r)
-		);
+		}).catch(() => {});
```
Comment (Floinko) [inline frontend/src/pages/EmailConfirmPage.js:22]: Why was this removed? Just ran into an error regarding the mail verification and had no information on that at all...

File: frontend/src/utils/utils.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Dominic Heil <d.heil@campus.tu-berlin.de>
+
+export const isValidEmail = (email) => {
+	const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```
Comment (Floinko) [inline frontend/src/utils/utils.js:5]: Please add your source for this regex here.

**Review by Floinko — COMMENTED** [review]
Code looks very good to me (except the two things i commented on) and everything worked for me when i tested it. However i think is still some work to do in terms of documentation:

- commenting in general
- how do i become the first admin? => create group "Admins" in appwrite
- which config do i have to change before starting the project? => currently harcoded for a local instance in config.js
- enable invites in appwrite

File: frontend/src/pages/EmailConfirmPage.js (hunk 1 above)
Comment (dominic-heil) [reply frontend/src/pages/EmailConfirmPage.js:22]: I added it back for now. I removed it as I thought we do not want to have anything written to the console when we deploy the frontend somewhere, but for now I think it makes sense to keep it in.

File: frontend/src/utils/utils.js (hunk 2 above)
Comment (dominic-heil) [reply frontend/src/utils/utils.js:6]: Done.

**Comment by dominic-heil** [comment]
@Floinko thank you for the feedback. I will add comments to each component/page later today, as well as update the readme in regards to setting up appwrite (i.e. linking to the backend scripts).

**Review by BerinikeTech — APPROVED** [review]
A little bit more documentation would definitely be helpful, but I think you did a great job naming the parameters, really straight forward and no room for misinterpretation

**Comment by Floinko** [comment]
LGTM!
